Dockerfile

```
@@ -3,7 +3,7 @@
 ARG NEXUS_VERSION=latest
 
 FROM maven:3-jdk-8-alpine AS build
```

```suggestion
FROM maven:3-eclipse-temurin-17 AS build
```

I'd suggest to drop the `apk add` in favor of switching from alpine to focal Maven image which already contains _git_ (little bigger, but the complete image can be cached)

We should also update it to JDK 17 to be consistent with the build pipeline, although 8 still works for now.

---

I think you are right, but this PR is just a bugfix, and should not introduce new "features".

---

I wouldn’t consider streamlining the outdated (2 years) build JDK with the main pipeline a "feature".

https://github.com/sonatype-nexus-community/nexus-repository-composer/blob/80fbc8748d923dc406680360ac0b90c7cac20b9c/.circleci/config.yml#L18

---

I’m not here to debate whether this should be called a feature, update, or enhancement. I just wanted to point out that it’s an unrelated change to the original PR and the specific bug it addresses.

As I’m new to this project, I mean no offense and don’t intend to tell anyone how to manage contributions. I just wanted to bring it to your attention. Thank you! :-)

---

It's a different solution for the problem.

The problem is: `maven:3-jdk-8-alpine` is not suited (any more) to build the project without modification.

While yes, installing `git` is a possible solution, it's still a bad one. `maven:3-jdk-8-alpine` is 5(!) years old: Alpine 3.9.4, OpenJDK 1.8.0u212, Maven 3.6.1

I don't really care if we switch to `3-eclipse-temurin-8-alpine` and install git, switch to `3-eclipse-temurin-8` or directly to `eclipse-temurin-17`, but any of these would be better than installing ancient git 2.20 in an outdated build environment.

---

You are right.